README.md

```diff
@@ -16,6 +16,22 @@ modules: {
 }
 ```
 
+You can specify properties to omit by setting an `omitFields` option. This option takes an array of field names. For example, if you wanted to exclude the `trash` field from the export, you'd configure your piece like this:
+
+```javascript
+modules: {
+  'apostrophe-pieces-export': {},
+  // other modules...
+  'my-module-that-extends-pieces': {
+    // Without this, there is no export functionality for this type.
+    // Not all types are great candidates for exports.
+    export: {
```

Comment:
This makes it sound like you *must* specify omitFields. I suggest giving both examples (just "true" and with omitFields).

Reply:
the example with just `true` is still there (further up in the readme)

Reply:
point taken about the inline comments though - removed for clarity (they were copied from the other example)